lib/sarif/bandit_sarif.rb

```diff
@@ -0,0 +1,57 @@
+module Sarif
+  class BanditSarif < BaseSarif
+    BANDIT_URI = 'https://github.com/PyCQA/bandit'.freeze
+
+    def initialize(scan_report)
+      super(scan_report)
+      @uri = BANDIT_URI
+      begin
+        @logs = JSON.parse(@scan_report.log(''))['results']
+      rescue JSON::ParserError
+        @logs = []
+      end
+      @unique_issues = Set.new
+    end
+
```

Comment:
Will the sarif report include errors?

Reply:
erorrs are logged in the invocations section